pkg/providers/launchtemplate/suite_test.go

```diff
@@ -1380,6 +1386,188 @@ var _ = Describe("LaunchTemplates", func() {
 				ExpectLaunchTemplatesCreatedWithUserData(expectedUserData)
 			})
 		})
+		Context("AL2023", func() {
+			BeforeEach(func() {
+				nodeClass.Spec.AMIFamily = &v1beta1.AMIFamilyAL2023
+
+				// base64 encoded version of "ca-bundle" to ensure the nodeadm bootstrap provider can decode successfully
+				awsEnv.LaunchTemplateProvider.CABundle = lo.ToPtr("Y2EtYnVuZGxlCg==")
+				options.FromContext(ctx).ClusterCIDR = "10.100.0.0/16"
+			})
+			Context("Kubelet", func() {
```

Comment:
If AL2023 is raid-0, we should have test for the new Ephemeral Storage calcualtion, no?

Reply:
Yep, still need to do the updates for the ephemeral storage calculation as well.